packages/core/src/main/Account.ts

```diff
@@ -290,10 +294,22 @@ export class Account extends EventEmitter {
     const loadedClient = await this.service!.client.getLocalClient();
     await this.apiClient.api.client.getClient(loadedClient.id);
     this.apiClient.context!.clientId = loadedClient.id;
+    if (this.enableMLS) {
+      this.coreCryptoClient = await this.createCoreCryptoClient(loadedClient);
+    }
 
     return loadedClient;
   }
 
+  private async createCoreCryptoClient(client: RegisteredClient): Promise<CoreCrypto> {
+    const {CoreCrypto} = await import('./CoreCrypto');
```

Comment:
Since the coreCrypto bundle might be quite big, we want to lazy load it, only if the consumer creates an mls client. 

Other users will never download the bundle